tools/lsp/preview/ui.rs

```diff
@@ -500,11 +498,20 @@ fn map_value_and_type(
             mapping.current_values.push(PropertyValue {
                 display_string: slint::format!("{}{}", get_value::<f32>(value), Unit::Phx),
                 kind: PropertyValueKind::Float,
+                value_kind: PropertyValueKind::Float,
                 value_float: get_value::<f32>(value),
-                visual_items: unit_model(&[Unit::Phx]),
-                value_int: 0,
+                visual_items: unit_model(&[
+                    Unit::Px,
+                    Unit::Cm,
+                    Unit::Mm,
+                    Unit::In,
+                    Unit::Pt,
+                    Unit::Phx,
+                    Unit::Rem,
+                ]),
+                value_int: 6,
                 code: get_code(value),
-                default_selection: 0,
+                default_selection: 6,
```

Comment:
did you mean `5` here? (for Phx)

Reply:
I changed the tests to not compare the index top be some expected value (which I looked up in the code setting the index;-).

I now compare the value in the array of Unit(types) at the index to the expected index. That catches the issue nicely.